### PR TITLE
NIFI-2816 - Clean typos across the code

### PR DESCRIPTION
**Pull request by trixpan**

### CODE REVIEW

**Comment by trixpan** [comment]
@pvillard31 second and final part of the typo clean up. Would appreciate a quick review so I don't need to spend eternity rebasing the PR.

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/provenance/provenance-content.jsp [hunk 1]
```diff
@@ -17,8 +17,8 @@
 <%@ page contentType="text/html" pageEncoding="UTF-8" session="false" %>
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <div id="provenance">
-    <span id="intial-component-query" class="hidden"><c:out value="${param.componentId}"/></span>
```
Comment (pvillard31) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/provenance/provenance-content.jsp]: Not related to some CSS file?

**Comment by trixpan** [comment]
@mcgillman would you mind having a look?

**Comment by trixpan** [comment]
@mcgilman would you mind having a look?

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/provenance/provenance-content.jsp (hunk 1 above)
Comment (trixpan) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/provenance/provenance-content.jsp]: Thanks for spotting this. Couldn't find a CSS but found a javascript file referencing to it. Added to a second commit.

**Comment by mcgilman** [comment]
Looks good @trixpan! Thanks this has been merged to master.

**Pull request closed without merging**